functions/Find-DbaStoredProcedure.ps1

```diff
@@ -132,9 +133,11 @@ Searches in "mydb" database stored procedures for "runtime" in the textbody
 					
 					foreach ($row in $rows)
 					{
-						$totalcount++; $sproccount++
+						$totalcount++; $sproccount++; $everyserverspcount++
 						
-						Write-Verbose "Looking in StoredProcedure: $proc TextBody for $pattern"
+						$proc = $row.name
```

Comment:
this $proc var doesn't seem to be used anywhere in the new code. Can it be removed?